modules/social_features/social_profile/src/SocialProfileTrait.php

```diff
@@ -122,20 +159,26 @@ private function startQuery() {
    *   The select query.
    */
   private function sortQuery(SelectInterface $query, $name, $suggestion_format) {
```

Comment:
Please add that to the `@param string $name` description that it is already sanitized.

Reply:
Is it enought?

Reply:
Yes, thanks!